### PR TITLE
fix: Fix warning for using jinja templates in assert

### PR DESCRIPTION
**Pull request by Jakuje**
Enhancement: Fix wrong assert in test

Reason: The assert was wrong causing warning and test failures on Debian

Result: The assert was fixed

Issue Tracker Tickets (Jira or BZ if any): -

### CODE REVIEW

**Review by spetrosi — APPROVED** [review]

**Comment by Jakuje** [comment]
looks like there is  more ...

**Comment by Jakuje** [comment]
For the last issue on the debian-buster, I filled roles-ansible/check-ansible-debian-buster-action#3

File: tests/tests_additional_packages.yml [hunk 1]
```diff
@@ -26,4 +26,4 @@
     - name: Verify the packages were installed
       assert:
         that:
-          - "'{{ additional_package | trim }}' in ansible_facts.packages"
+          - "additional_package | trim in ansible_facts.packages"
```
Comment (richm) [inline tests/tests_additional_packages.yml:29]: note that you do not need quotes here - this should work
```
          - additional_package | trim in ansible_facts.packages
```
same with the below

Comment (Jakuje) [reply tests/tests_additional_packages.yml]: Thanks! Updated.

File: tests/tests_all_options.yml [hunk 2]
```diff
@@ -85,6 +85,6 @@
     - name: Verify the options are in the file
       assert:
         that:
-          - "'{{ item }} yes' in config.content | b64decode "
+          - "item ~ ' yes' in config.content | b64decode "
```
[the remark this answers is not in the document]
Comment (Jakuje) [reply tests/tests_all_options.yml]: removed. Thanks.

File: tests/tests_additional_packages.yml (hunk 1 above)
Comment (Jakuje) [reply tests/tests_additional_packages.yml]: sounds like this does not work, at least not for the more complicated expressions as in `tests/tests_no_skip_defaults.yml`. But dropped the needless quotes from the rest of the ones.

**Comment by Jakuje** [comment]
> @Jakuje is this ready to merge?

Yes. Please merge.

**Review by richm — APPROVED** [review]

**Pull request merged**